workbench-core/accounts/src/services/accountService.ts

```diff
@@ -61,14 +62,12 @@ export default class AccountService {
       index: 'getResourceByCreatedAt',
       key: { name: 'resourceType', value: 'account' }
     };
-    const response = await this._aws.helpers.ddb.query(queryParams).execute();
-    let accounts: Account[] = [];
-    if (response && response.Items) {
-      accounts = response.Items.map((item: unknown) => {
-        return item as unknown as Account;
-      });
-    }
-    return accounts;
+
+    const response = await this._aws.helpers.ddb.getPaginatedItems(queryParams);
+
+    return response.data.map((item) => {
+      return AccountSchema.parse(item);
```

Comment:
Ah is this why we need `accountSchema`? So we can `.parse` a zod onject instead of casting `as unknown as Account` using the existing model?

Reply:
We use the Zod parser to parse the `item` that comes back from `getPaginatedItems`. It validates AND parses the data for us, guaranteeing that TSC is getting the static type correct and that our transpiled Javascript code is getting the types we expect at runtime.